src/main/java/mate/controller/AddDriverController.java

```diff
@@ -0,0 +1,30 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class AddDriverController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final DriverService driverService = (DriverService)
+            injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/drivers/create.jsp").forward(req, resp);

```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/drivers/add.jsp").forward(req, resp);
```
rename all create jsps